### PR TITLE
Add styling to AddNodeModal

### PR DESCRIPTION
**Pull request by dimaip**

### CODE REVIEW

**Comment by grebaldi** [comment]
:+1:

ping @Inkdpixels

File: Resources/Private/JavaScript/Components/Dialog/index.js [hunk 1]
```diff
@@ -30,6 +31,9 @@ const Dialog = props => {
                             id="neos__modal__closeModal"
                             onClick={e => executeCallback({e, cb: onRequestClose})}
                             />
+                        <div className={style.dialog__title}>
```
Comment (Inkdpixels) [inline Resources/Private/JavaScript/Components/Dialog/index.js]: Looks good, but have you considered using a <h1> (or lower) tag for this?

File: Resources/Private/JavaScript/Components/Dialog/style.css [hunk 2]
```diff
@@ -16,8 +16,12 @@
 }
 .dialog__contents {
     position: relative;
-    width: 80vw;
-    min-height: 300px;
+    width: 50vw;
```
Comment (Inkdpixels) [inline Resources/Private/JavaScript/Components/Dialog/style.css]: @aertmann posted this in slack, just as a reminder :) :+1:
https://github.com/neos/neos-development-collection/commit/dd81235f0bb9cb455867ff182e6db42617e138ab

File: Resources/Private/JavaScript/Host/Containers/AddNodeModal/index.js [hunk 3]
```diff
@@ -86,10 +86,10 @@ export default class AddNodeModal extends Component {
     renderNodeTypeGroup(group, key) {
         return (
             <div key={key}>
-                <Headline type="h2">
+                <div className={style.groupTitle}>
```
Comment (Inkdpixels) [inline Resources/Private/JavaScript/Host/Containers/AddNodeModal/index.js]: Also, why not a heading with additional styling for semantics?

File: Resources/Private/JavaScript/Components/Dialog/index.js (hunk 1 above)
Comment (dimaip) [inline Resources/Private/JavaScript/Components/Dialog/index.js]: Yes, I will switch to it. I want to collect all typography styles from current neos interface, and assign them to h1-h6 Headline components, would that be a way to go?

File: Resources/Private/JavaScript/Components/Dialog/style.css (hunk 2 above)
Comment (dimaip) [inline Resources/Private/JavaScript/Components/Dialog/style.css]: Thanks for reminder, was planning to do it, but forgot :)

**Comment by dimaip** [comment]
@Inkdpixels hey, I've adjusted the width and height to new values, and also created a wider variant of the Dialog, according to how it was in current design.
I want to change a lot of things how we handle styling in general, and will create a proposal for that, and will take care of that refactoring during the sprint, but for now I propose to merge this one as is.

**Pull request merged**